cmd/list.go

```diff
@@ -1243,6 +1244,76 @@ var listOrganizationsCmd = &cobra.Command{
 	},
 }
 
+var listEnvironmentServicesCmd = &cobra.Command{
+	Use:     "environment-services",
+	Aliases: []string{"e"},
```

Comment:
Alias is the same as `listEnvironmentsCmd` - maybe update it to `es`? Works well otherwise!

Reply:
Ah nice spot. I wonder if there is a way to detect this easier